README.adoc

```diff
@@ -380,17 +377,15 @@ For more information about how to use the API or to customize the output, refer
 
 == Contributing
 
-In the spirit of {uri-freesoftware}[free software], _everyone_ is encouraged to help improve this project.
-If you discover errors or omissions in the source code, documentation, or website content, please don't hesitate to submit an issue or open a pull request with a fix.
 New contributors are always welcome!
+If you discover errors or omissions in the source code, documentation, or website content, please don't hesitate to submit an issue or open a pull request with a fix.
 
 Here are some ways *you* can contribute:
 
 * by using prerelease (alpha, beta or preview) versions
 * by reporting bugs
 * by suggesting new features
 * by writing or editing documentation
-* by writing specifications
 * by writing code -- _No patch is too small._
```

Comment:
I agree that "writing specifications" was too vague. Let's also change the "writing code" line to

```
* by writing code with tests
```